content/tutorial/tutorial.md

```diff
@@ -740,7 +740,7 @@ We can now change the Board's `handleClick` function to return early by ignoring
 ```javascript{3-5}
   handleClick(i) {
     const squares = this.state.squares.slice();
-    if (calculateWinner(squares) || squares[i]) {
+    if (squares[i] || calculateWinner(squares)) {
```

Comment:
It would be inconsistent if we change it in the docs alone. There are multiple codepen links where we might need to change as well?.

Reply:
Sure, should be changed in the `codepen` also. Just that I don't think that is managed in this repo. I would if I have access to the online codes, I guess the admins need to do that.

Reply:
Changing in codepen might require Dan's help. Let's wait for his thoughts on this change as well.